core/arch/arm/plat-sam/sama7g5.h

```diff
@@ -0,0 +1,400 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Header file for ATSAMA7G54
+ *
+ * Copyright (c) 2023 Microchip Technology Inc. and its subsidiaries.
+ *
+ * Subject to your compliance with these terms, you
+ * may use Microchip software and any derivatives
+ * exclusively with Microchip products. It is your
+ * responsibility to comply with third party license
+ * terms applicable to your use of third party software
+ * (including open source software) that may
+ * accompany Microchip software.
+ *
+ * THIS SOFTWARE IS SUPPLIED BY MICROCHIP "AS IS".
+ * NO WARRANTIES, WHETHER EXPRESS, IMPLIED OR STATUTORY,
+ * APPLY TO THIS SOFTWARE, INCLUDING ANY IMPLIED WARRANTIES
+ * OF NON-INFRINGEMENT, MERCHANTABILITY, AND
+ * FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * IN NO EVENT WILL MICROCHIP BE LIABLE FOR ANY INDIRECT,
+ * SPECIAL, PUNITIVE, INCIDENTAL OR CONSEQUENTIAL
+ * LOSS, DAMAGE, COST OR EXPENSE OF ANY KIND WHATSOEVER
+ * RELATED TO THE SOFTWARE, HOWEVER CAUSED, EVEN IF
+ * MICROCHIP HAS BEEN ADVISED OF THE POSSIBILITY OR
+ * THE DAMAGES ARE FORESEEABLE. TO THE FULLEST EXTENT
+ * ALLOWED BY LAW, MICROCHIP'S TOTAL LIABILITY ON ALL
+ * CLAIMS IN ANY WAY RELATED TO THIS SOFTWARE WILL NOT
+ * EXCEED THE AMOUNT OF FEES, IF ANY, THAT YOU HAVE
+ * PAID DIRECTLY TO MICROCHIP FOR THIS SOFTWARE.
+ *
+ */
+
+/* File generated from device description version 2023-02-02T10:07:10Z */
+#ifndef _SAMA7G54_H_
+#define _SAMA7G54_H_
+
+/* Header version uses Semantic Versioning 2.0.0 (https://semver.org/) */
+#define HEADER_FORMAT_VERSION "2.1.0"
```

Comment:
Do we need all this unused stuff?
OP-TEE OS doesn't use C++ either.

Reply:
Hi @jenswi-linaro ,
The header file updated by removing the unused stuff, thank you for your comments.